utils/collectd/Makefile

```diff
@@ -501,5 +527,5 @@ $(eval $(call BuildPlugin,vmem,virtual memory usage input,vmem,))
 $(eval $(call BuildPlugin,wireless,wireless status input,wireless,))
 $(eval $(call BuildPlugin,write-graphite,Carbon/Graphite output,write_graphite,+PACKAGE_collectd-mod-write-graphite:libpthread))
 $(eval $(call BuildPlugin,write-http,HTTP POST output,write_http,+PACKAGE_collectd-mod-write-http:libcurl))
-
 $(eval $(call BuildScriptPlugin,sqm,SQM/qdisc collection,sqm_collectd,+PACKAGE_collectd-mod-sqm:collectd-mod-exec))
+$(eval $(call BuildScriptLuaPlugin,ltq-dsl,Lantiq DSL collection,dslstat,+PACKAGE_collectd-mod-ltq-dsl:collectd-mod-lua +luci-lib-nixio))
```

Comment:
I pulled this into my staging tree and noticed that this packages makes only sense for lantiq xrx200 targets.
So I think we should update the dependency with the additional target dependencies `@TARGET_lantiq_xrx200`

I think we also need to get rid of the dependency against `luci-lib-nixio` in the lua script.
The problem here I see is, that there are people who don't use the LuCI feed and so do not have this package.

We only need this for `local hostname = fs.readfile(hostname_file)` could you have a look to replace this with lua std lib calls ?
https://www.tutorialspoint.com/lua/lua_file_io.htm

Reply:
I managed to remove the `luci-lib-nixio` dependency now. Also I added the dependency on the `xrx200` target, it seems to work fine in the build menu.